scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -992,9 +991,9 @@
 (timers/deftimer [cook-mesos scheduler killing-cancelled-tasks-duration])
 
 (defn cancelled-task-killer
```

Comment:
I agree, but don't want to hold up the PR on them. My reasoning is that these functions do very little except call the functions that do real work (and are unit tested). Also, the main ones (rank, rebalance, match) are covered by the mesos mock cook scheduler integration test. Are you ok putting this off?